src/test/java/org/gaul/modernizer_maven_plugin/ModernizerTest.java

```diff
@@ -483,6 +487,11 @@ public Void apply(Void input) {
         public boolean apply(Void input) {
             return true;
         }
+
+        @Override
+        public boolean test(Void input) {
+            return true;
+        }
```

Comment:
This is a side-effect of the Guava upgrade (would also be necessary i.c.w. version 21.0.)